src/qml/editorwidgets/ValueMap.qml

```diff
@@ -272,7 +272,6 @@ EditorWidgetBase {
                 font: parent.font
                 width: parent.width
                 verticalAlignment: Text.AlignVCenter
-                leftPadding: parent.indicator.width + parent.spacing
```

Comment:
Has this changed the way the widget looks? While it might look chaotic in there, quite a few tweaks are in place to insure consistency across the widgets in terms of spacing.

Reply:
I wanted to make a try because the linter was complaining with `Invalid property name "leftPadding" (M16)`. So I removed to see if there is some difference.
Now I have seen that by importing QtQuick 2.15 instead of 2.14 the warning goes away. Should I lift the version everywhere?
I have seen that since Qt6 the import version can be left away. Should we do that or is better to specify it?

Reply:
@domi4484 , bump to 2.15 for now here. If you feel like it, in a different PR, go ahead and remove all versions from all of our QML files, it's something I've been wanting to do :)

Reply:
@nirvn btw unreleated to that i noticed some margin issue in the latest pr builds (left pr builds, right official release):
![image1](https://github.com/opengisch/QField/assets/9881900/2dd18a4a-1318-4eb5-9b59-77d3619dd7dc)

Reply:
@domi4484 , https://github.com/opengisch/QField/pull/5207